contrib/automation_tests/test_cases/capture_window.py

```diff
@@ -331,7 +331,7 @@ def _set_collect_thread_states_capture_option(self, collect_thread_states: bool,
                                                            'CollectThreadStatesCheckBox',
                                                            parent=capture_options_dialog)
         if collect_thread_states_checkbox.get_toggle_state() != collect_thread_states:
-            logging.info('Toggling "Collect thread states" checkbox')
+            logging.info('Setting "Collect thread states" checkbox to {}.', collect_thread_states)
```

Comment:
I like this. Much clear than before. 

But I saw you sometimes also use "Toggling" for the memory and manual instrumentation checkboxes. It would be better to be consistently using either "Setting" or "Toggling"

Reply:
I prefer "Toggling" : )

Reply:
done!